pkg/inventory/model/model_test.go

```diff
@@ -85,7 +85,7 @@ func TestModels(t *testing.T) {
 		ListOptions{
 			Labels: Labels{
 				"role": "main",
-		}},
+			}},
```

Comment:
This is just a `go fmt` change that got missed in a previous PR.  Figured it would be fine to leave it in.